### PR TITLE
feat: Add reference information for volume types

### PR DESCRIPTION
**Pull request by fghaas**
In the reference section, add a new subsection about volume types. Explain the `cbs` and `cbs-encrypted` volume types, their encryption support, and quality-of-service (QoS) caps.

### CODE REVIEW

File: docs/reference/index.md [hunk 1]
```diff
@@ -1,31 +1,22 @@
...
+
+* Our **[Service Version Matrix](versions/index.md)** lists the open source software versions running in our {{brand}} regions.
 
 ## API reference
 
-* The **[API reference](api/index.md)** documentation provides useful
-  information for accessing Application Programming Interfaces (APIs),
-  getting information, and manipulating all sorts of {{brand}}
-  objects.
+* The **[API reference](api/index.md)** documentation provides useful information for accessing Application Programming Interfaces (APIs), getting information, and manipulating all sorts of {{brand}}  objects.
```
Comment (colder-is-better) [inline docs/reference/index.md:19]: There is this double-space after {{brand}}...

File: docs/reference/volumes/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Volume types
+
+The following volume types are available in {{brand}} for persistent block storage devices ("volumes") managed by OpenStack Cinder.
+
+If you create a volume without a specified volume type, then the default volume type applies.
```
Comment (colder-is-better) [inline docs/reference/volumes/index.md:5]: Avoid the passive voice?

```suggestion
If you create a volume without specifying a volume type, then the default volume type applies.
```

**Review by colder-is-better — COMMENTED** [review]
All is looking good, I only have a couple of comments/suggestions.

File: docs/reference/index.md (hunk 1 above)
Comment (fghaas) [reply docs/reference/index.md]: Good catch, thanks. Fixed.

**Review by colder-is-better — APPROVED** [review]
Allt gott!

**Comment by fghaas** [comment]
@noonedeadpunk Can you give this one an extra pair of eyes, please?

**Review by colder-is-better — APPROVED** [review]
LGTM!

**Review by noonedeadpunk — APPROVED** [review]
LGTM

**Comment by fghaas** [comment]
@colder-is-better Looks like we can finally merge this one, so I rebased on current `main` and we'll need one more approval please.

**Review by colder-is-better — APPROVED** [review]

**Pull request merged**